packages/config/src/base.js

```diff
@@ -21,11 +22,13 @@ const getInitialBase = function ({
 //  - To resolve file paths
 // If the second file has a `base` property, it is ignored, i.e. it is not
 // recursive.
-// Also add it to `config.build.base`.
-const addBase = function (repositoryRoot, config) {
-  const base = resolvePath(repositoryRoot, config.build.base)
-  const configA = { ...config, build: { ...config.build, base } }
-  return { base, config: configA }
+const getBase = function (base, repositoryRoot, config) {
+  return base === undefined ? resolvePath(repositoryRoot, config.build.base) : base
```

Comment:
On the first round, `base` is `undefined`. On the second, it is defined, and we keep it as is, as opposed to using the second `netlify.toml` `base` property, since that one is never used (because there is no third round).